test/AddrArrayLib-test.js

```diff
@@ -0,0 +1,60 @@
+const TokenContract = artifacts.require("TokenContract");
+
+contract('TokenContract', (accounts) => {
+
+  const owner = accounts[0];
+  let tokenContractInsatnce;
+
+  it('should deploy the smart contrat and check total supply', async () => {
```

Comment:
should deploy the smart contrat and check total supply -> should deploy the smart contract and set the correct initial total supply